plugins/modules/database/misc/kibana_plugin.py

```diff
@@ -222,7 +237,8 @@ def main():
             plugin_bin=dict(default="/opt/kibana/bin/kibana", type="path"),
             plugin_dir=dict(default="/opt/kibana/installedPlugins/", type="path"),
             version=dict(default=None),
-            force=dict(default="no", type="bool")
+            force=dict(default="no", type="bool"),
+            allow_root=dict(default="no", type="bool"),
```

Comment:
```suggestion
            force=dict(default=False, type="bool"),
            allow_root=dict(default=False, type="bool"),
```

Reply:
Is it alright to leave force as it is for now, since this is not my addition?